### PR TITLE
gh-104679 Fixed syntax highlighting in turtle docs

### LINKED ISSUE #104679: Syntax highlighting issue in Turtle docs
# Documentation

A couple of function docs

- https://docs.python.org/3/library/turtle.html#turtle.setposition
- https://docs.python.org/3/library/turtle.html#turtle.pencolor
- https://docs.python.org/3/library/turtle.html#turtle.fillcolor
- https://docs.python.org/3/library/turtle.html#turtle.color
- https://docs.python.org/3/library/turtle.html#turtle.filling
- https://docs.python.org/3/library/turtle.html#turtle.shearfactor

image

seems to not have syntax highlighting working properly. 
On inspection, I see this was due to small indentation issue.

### Linked PRs
* gh-104682
* gh-104695

### PR DESCRIPTION
**Pull request by millefalcon**
Fixed indentation for the code snippets in `turtle` library docs which was causing syntax highlighting issues

closes #104679

----
:books: Documentation preview :books:: https://cpython-previews--104682.org.readthedocs.build/

* Issue: gh-104679

### CODE REVIEW

File: Doc/library/turtle.rst [hunk 1]
```diff
@@ -2023,19 +2023,19 @@ Settings and special methods
...
    (2) *name* is an arbitrary string and *shape* is a tuple of pairs of
        coordinates: Install the corresponding polygon shape.
 
-       .. doctest::
-          :skipif: _tkinter is None
+      .. doctest::
+         :skipif: _tkinter is None
 
-          >>> screen.register_shape("triangle", ((5,-3), (0,5), (-5,-3)))
+         >>> screen.register_shape("triangle", ((5,-3), (0,5), (-5,-3)))
```
Comment (hugovk) [inline Doc/library/turtle.rst]: I think the indent might have been correct before? The HTML looks better nested.

Before: https://docs.python.org/3.12/library/turtle.html#turtle.addshape

image

After: https://cpython-previews--104682.org.readthedocs.build/en/104682/library/turtle.html#turtle.addshape

image

**Review by hugovk — COMMENTED** [review]
Thanks! Mostly looks good, one suggestion.

File: Doc/library/turtle.rst (hunk 1 above)
Comment (millefalcon) [reply Doc/library/turtle.rst]: Oh, right. My bad. Will fix soon

Comment (millefalcon) [reply Doc/library/turtle.rst]: @hugovk Fixed. Is that fine ? 😅

Comment (millefalcon) [reply Doc/library/turtle.rst]: image

Comment (hugovk) [reply Doc/library/turtle.rst]: Thank you!

**Comment by hugovk** [comment]
@millefalcon Would you like to make the same change for the `3.11` branch? (No requirement to use `cherry_picker`.)

**Comment by millefalcon** [comment]
@hugovk Sure. Let me check what one needs to do for targetting a specific branch. Thanks 👍

File: Doc/library/turtle.rst (hunk 1 above)
Comment (millefalcon) [reply Doc/library/turtle.rst]: No problem 👍 and thank you

**Pull request merged**